libraries/aws/provisioning/include/types/aws_iot_provisioning_types.h

```diff
@@ -279,15 +279,15 @@ typedef struct AwsIotProvisioningRejectedResponse
 
 /**
  * @ingroup provisioning_datatypes_paramstructs
- * @brief Aggregates the data sent as response from AWS IoT Core for the request to create a new certificate for a CSR request.
+ * @brief Aggregates the data from AWS IoT Core's response to the certificate creation request with a CSR.
  *
  * @paramfor Response Callback function of @ref provisioning_function_createkeysandcertificate
  *
  * The @ref AwsIotProvisioning_CreateKeysAndCertificate library API passes this object to a user-provided callback
  * function
  * whenever the operation completes with a response from the server.
  */
-typedef struct AwsIotProvisioningCreateCertificateFromCsrResponse
+typedef struct AwsIotProvisioningCreateCertFromCsrResponse
```

Comment:
This change can also apply to `AwsIotProvisioningCreateCertificateFromCsrCallbackInfo_t` and the functions with long names as well.